src/views/ProductView.vue

```diff
@@ -1,21 +1,87 @@
 <template>
     <div class="d-flex flex-column ga-4">
         <div>
-            <v-toolbar class="bg-grey-lighten-4" density="compact">
+            <v-toolbar class="bg-grey-lighten-3" density="comfortable">
                 <v-btn icon="mdi-arrow-left" @click="router.back()"></v-btn>
-                <v-spacer></v-spacer>
             </v-toolbar>
+            <v-card class="bg-grey-lighten-3" rounded="0" variant="flat">
+                <v-card-item>
+                    <v-card-title class="mr-1">
+                        Product: {{ product?.product.internalName }}
+                    </v-card-title>
+
+                    <v-card-subtitle
```

Comment:
This formatting looks a little sus

Reply:
Maybe a manual break after the > makes it look better without breaking anything frontend related?